saleor/graphql/webhook/subscription_payload.py

```diff
@@ -56,7 +56,8 @@ def initialize_context() -> HttpRequest:
     handler = BaseHandler()
     context = RequestFactory().request()
     handler.load_middleware()
-    handler.get_response(context)
+    response = handler.get_response(context)
+    assert response.status_code == 200
```

Comment:
How does the assert fixes the issue? Shouldn't we somehow handle this error?

Reply:
The issue is if the error happens during the fixture, we never get to notice it. Thus we can get unexpected test results, where with this change we can see clearly that the status code error wasn't 200.

There isn't an issue with Saleor but the issue can happen in forks where there is additional logic